src/main/java/uk/gov/companieshouse/web/accounts/model/smallfull/CashAtBankAndInHand.java

```diff
@@ -2,12 +2,16 @@
 
 import lombok.Getter;
 import lombok.Setter;
+import uk.gov.companieshouse.web.accounts.validation.ValidationMapping;
 
 @Getter
 @Setter
 public class CashAtBankAndInHand {
 
+    @ValidationMapping("$.current_period.balance_sheet.current_assets.cash_at_bank_and_in_hand")
```

Comment:
You'll need to add a `@ValidationModel` annotation to this class for these mappings to be picked up.

Reply:
I didn't think that was the case? The scanner works recursively from the top level model, and BalanceSheet holds the `@ValidationModel` annotation.